src/keria/app/agenting.py

```diff
@@ -889,31 +888,34 @@ def on_get(req, rep, alias):
         if role in (kering.Roles.witness,):  # Fetch URL OOBIs for all witnesses
             oobis = []
             for wit in hab.kever.wits:
-                urls = hab.fetchUrls(eid=wit, scheme=kering.Schemes.http)
+                urls = hab.fetchUrls(eid=wit, scheme=kering.Schemes.http) or hab.fetchUrls(eid=wit, scheme=kering.Schemes.https)
```

Comment:
should add a test to cover this branch in logic

Reply:
Agreed. We don't have coverage of that branch. In order to fully test that branch it seems we need to bring in locally self signed certificates for proper TLS. I could bring that in from HIO like I mentioned. Would you still prefer mocking in this case? We'd just need a hab with TLS urls. I'm not familiar with the test fixtures though I suppose there is a Hab test fixture pretty close to what we need.

Reply:
Yeah, looks like we'll need a new `helpers.openKeria()` test fixture that returns an agency started up with TLS config. Want me to make that for this PR?